theme-check.rb

```diff
@@ -0,0 +1,109 @@
+# -*- ruby -*-
+# autogenerated via `brew gem formula theme-check`
+# https://github.com/sportngin/brew-gem
+
+require 'formula'
+require 'fileutils'
+
+USE_HOMEBREW_RUBY = true
+
+module RubyBin
+  def ruby_bin
+    USE_HOMEBREW_RUBY ? Formula["ruby"].opt_bin: "/usr/bin"
+  end
+end
+
+class RubyGemsDownloadStrategy < AbstractDownloadStrategy
+  include RubyBin
+
+  def fetch
+    ohai "Fetching theme-check from gem source"
```

Comment:
Is this standard? I don't see any other formula using this.

Reply:
I think it's used from within Homebrew. It creates the emboldened lines when downloading:
![](https://screenshot.click/19-03-b8whn-gudik.png)